packages/react-storage/src/components/StorageBrowser/displayText/libraries/en/uploadView.ts

```diff
@@ -10,74 +21,53 @@ export const DEFAULT_UPLOAD_VIEW_DISPLAY_TEXT: DefaultUploadViewDisplayText = {
   addFolderLabel: 'Add folder',
   getActionCompleteMessage: (data) => {
     const { counts } = data ?? {};
-    const { COMPLETE, FAILED, OVERWRITE_PREVENTED } = counts ?? {};
+    const { COMPLETE, FAILED, OVERWRITE_PREVENTED, CANCELED, TOTAL } =
+      counts ?? {};
 
     const hasPreventedOverwrite = !!OVERWRITE_PREVENTED;
     const hasFailure = !!FAILED;
     const hasSuccess = !!COMPLETE;
-
-    const preventedOverwriteMessage = !hasPreventedOverwrite
-      ? undefined
-      : `Overwrite prevented for ${OVERWRITE_PREVENTED} file${
-          OVERWRITE_PREVENTED > 1 ? 's' : ''
-        }`;
-
-    const failedMessage = !hasFailure
-      ? undefined
-      : `${FAILED} file${FAILED > 1 ? 's' : ''} failed to upload`;
-
-    const completedMessage = !hasSuccess
-      ? undefined
-      : `${COMPLETE} file${COMPLETE > 1 ? 's' : ''} uploaded`;
+    const hasCanceled = !!CANCELED;
 
     const type = hasFailure
       ? 'error'
-      : hasPreventedOverwrite
+      : hasPreventedOverwrite || hasCanceled
       ? 'warning'
       : 'success';
```

Comment:
Simllar to the comment provided on line 5, don't think we should be using a util, providing the specific `type` in the each case is intentional